README.md

```diff
@@ -97,9 +94,14 @@ use to talk to the unikernel:
 
 **Running with ukvm on Linux**
 
+A specialized monitor called `ukvm-bin` is generated as part of the
+build process, so the command line arguments may differ depending on
+the needs of the unikernel.  To see the arguments, run `ukvm-bin` with
+no arguments or `--help`.
+
 To launch the unikernel:
 
-    ukvm /dev/zero tap100 ./mir-stackv4.ukvm
+    ukvm-bin --net=tap100 ./mir-stackv4.ukvm
```

Comment:
Should be `./ukvm-bin` (unless you have `.` in `$PATH` which is generally considered a bad idea).